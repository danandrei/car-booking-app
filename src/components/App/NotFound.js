import React from 'react';

const NotFound = () => (
  <div className="text-center cars-wrap">
    <h1>404! Not found.</h1>
  </div>
);

export default NotFound;
