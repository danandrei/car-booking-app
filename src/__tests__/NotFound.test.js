import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });
import { shallowToJson } from 'enzyme-to-json';

import NotFound from '../components/App/NotFound';

describe('NotFound', () => {
  it ('should render correctly', () => {
    const output = shallow(
      <NotFound />
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
