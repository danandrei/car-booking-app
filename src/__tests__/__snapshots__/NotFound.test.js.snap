// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render correctly 1`] = `
<div
  className="text-center cars-wrap"
>
  <h1>
    404! Not found.
  </h1>
</div>
`;
